test/js/factory/user.spec.js

```diff
@@ -0,0 +1,11 @@
+describe("User", function() {
```

Comment:
if you check the `User` in our code
it is written as a factory with a constructor
so it need to initialize it as a new object e.g. `user = new User()`